### ISSUE TITLE
Commit failed - exit code 128 received, with output: 'fatal: cannot update the ref 'refs/heads/master': unable to append to '.git/logs/refs/heads/master': Invalid argument'

### ISSUE THREAD

**Issue by cmcculloughdieter** [desktop/desktop#11295]
### Version & OS

Open 'About GitHub Desktop' menu to see the Desktop version. Also include what operating system you are using.

### Actual behavior

A clear and concise description of what actually happened.

### Screenshots

Add screenshots to help explain your problem, if applicable.

### Logs

Attach your logs by opening the `Help` menu and selecting `Show Logs...`, if applicable.

**Comment by cmcculloughdieter** [comment]
I do not know what to do to resolve this issue. I am not able to commit new files to this task in GitHub Desktop. I get this error message: Commit failed - exit code 128 received, with output: 'fatal: cannot update the ref 'refs/heads/master': unable to append to '.git/logs/refs/heads/master': Invalid argument'

**Comment by steveward** [comment]
@cmcculloughdieter this sounds like a permissions error. Can you verify that the correct permissions are set for your repository? What operating system are you using?

**Comment by cmcculloughdieter** [comment]
Thanks for replying. I was able to fix the problem by moving the bad files to a different location, then dropping and re-cloning the repository. Then I moved the files back into the desktop repository. This fixed the issue. I think it was caused by me using OneDrive from two different laptops. I have learned that this does not work very well for GitHub repository files. I now stick to a single laptop as the one that pushes data from my OneDrive to the github master online.


Carol McCullough-Dieter
Database Administrator
VAS
[cid:image001.png@01D6DDF1.47E16920]
Carol.McCullough-Dieter@vas.com<mailto:Carol.McCullough-Dieter@vas.com>
AgSource | 2418 Crossroads Drive, Suite 3400
Madison WI 53718 | USA
